src/services/helper/form.js

```diff
@@ -51,20 +52,18 @@ module.exports = class FormsHelper {
 	static async update(id, bodyData) {
 		try {
 			let filter = {}
-			if (ObjectId.isValid(id)) {
+			if (id) {
 				filter = {
-					_id: ObjectId(id),
+					id: id,
 				}
 			} else {
 				filter = {
 					type: bodyData.type,
```

Comment:
Note to self: Same pattern in user service. 

To dev: Since we are using form versions. Shouldn't it be part of the filter to identify which is the form that is actually getting updated in-case there is no id?

Reply:
Comment [Nevil]: The previous version isn't stored but gets overwritten.